### ISSUE TITLE
A11y_Azure Data Studio_Table_Columns_ScreenReader: The column header for the 6th column has not been provided. The voiceover is reading it as 'blank, column 6 of 6' while using keyboard arrow keys to move across the table.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19562]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioforMAC_May2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG4.2.3;#Voiceover;#AINotInScope;#Regressed:06-17-22;

## Environment Details:
Application Name: Azure Data Studio
Application Version: 1.36.1
Mac Version: macOS Monterey 12.4
Screen Reader: Voiceover

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure Data Studio and connect to server.
3. Tab to 'SERVERS' section and expand.
4. Tab till 'Databases' menu item and expand.
5. Select any of the databases and expand.
6. Tab till 'Tables' sub-menu item.
7. Right click on table sub-menu item and enter 'New Table'.
8. Tab till 'Columns' tab.
9. Turn on voiceover and tab to 6th column header of the table.

## Actual:
The column header for the 6th column has not been provided for the table under 'Columns' tab. The voiceover is reading it as 'blank, column 6 of 6' while using arrow keys to move across the table.

The same issue can be observed for all the tables in the application.

## Expected:
The column header for the 6th column should be provided for the table under 'Columns' tab. Here we can use 'Remove, column 6 of 6' for the voiceover to announce while using arrow keys to move across the table.

## User Impact:
Screen reader users will find it difficult to identify the column content if not announced properly.

## Attachment:
![2_Column header](https://user-images.githubusercontent.com/98393966/170928168-d1e76d5b-cb6a-49d4-bfb2-04c3524d0f80.gif)

**Comment by bbarik06** [comment]
We suggested you add a column header for column 6 which the voiceover was announcing as 'blank, column 6 of 6', Instead 2 new column headers (1st, 7th) have been added which voiceover is still announcing as 'Blank'. Please fix this issue by programmatically and visually assigning column headers. 

Issue is repro. Hence, reactivating the bug. Please refer the attachment.

![19562_Not Fixed](https://user-images.githubusercontent.com/98393966/174245480-e46ff803-4d92-460a-ad73-ec302ccf5b76.gif)

 **Note:** @alanrenmsft  Please do not close the bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix. Please reopen this until fixed completely.

**Comment by alanrenmsft** [comment]
@bbarik06 I just merged the fix for this issue, please use the latest insider build (at the earliest 6/21's insider release), I've added header for the columns. 

![image](https://user-images.githubusercontent.com/13777222/174687288-3093e101-b3aa-4562-87ac-92f875eef317.png)

**Comment by alanrenmsft** [comment]
@bbarik06 please let me know when you will be able to re do the test. thanks

**Comment by bbarik06** [comment]
@alanrenmsft Please provide the latest build to test where fixes have been made.

**Comment by alanrenmsft** [comment]
@bbarik06 https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main, you can use the latest insider build

**Comment by alanrenmsft** [comment]
@bbarik06 ping...

**Comment by sspatil123** [comment]
![image](https://user-images.githubusercontent.com/85863999/218399075-4770b38b-564b-4f72-9806-5907be103bc9.png)
GithubTags:#Closed;Version 1.41.1
